docs/user_guide/backend_configuration.rst

```diff
@@ -0,0 +1,286 @@
+Backend Configuration
+=====================
+
+NeuroConv offers highly convenient control over the type of file backend and the way its datasets are configured.
+
+Find out more about possible file types in the `main NWB documentation <https://nwb-overview.readthedocs.io/en/latest/faq_details/why_hdf5.html#why-use-hdf5-as-the-primary-backend-for-nwb>`_.
+
+Find out more about chunking and compression in the `advanced NWB tutorials for dataset I/O settings <https://pynwb.readthedocs.io/en/stable/tutorials/advanced_io/h5dataio.html#sphx-glr-tutorials-advanced-io-h5dataio-py>`_.
+
+Find out more about memory buffering of large source files in the `advanced NWB tutorials for iterative data write <https://pynwb.readthedocs.io/en/stable/tutorials/advanced_io/plot_iterative_write.html#sphx-glr-tutorials-advanced-io-plot-iterative-write-py>`_.
+
+
+
+Default configuration
+---------------------
+
+To retrieve a default configuration for an in-memory ``pynwb.NWBFile`` object, use the ``get_default_backend_configuration`` function...
```

Comment:
I would first have a code block declaring the nwbfile as a working example, then this comment, and then the final code block:

```
    default_backend_configuration = get_default_backend_configuration(
        nwbfile=nwbfile, backend="hdf5"
    )
```

Otherwise, users have to look for the `get_default_backend_configuration` line which is almost at the end. In other words, if you are introducing this function it should come shortly after you talk about it. What do you think?

Reply:
I think I will do this in general, that is, in the other sections I will create an example and say that we are working with this data, and then have a small code block where I only have the function that I want to explain.

One downside of my proposal is that is harder to copy paste but I am thinking this as an explainer rather than as a how-to so I think this makes sense.

Reply:
OK well see what it looks like in followup